craft_application/commands/lifecycle.py

```diff
@@ -524,9 +524,8 @@ def _run(
 
         if parsed_args.test_path:
             testing_service.validate_tests(parsed_args.test_path)
-        # Output into the spread directory.
-        parsed_args.output = pathlib.Path.cwd() / "spread"
-        parsed_args.output.mkdir(exist_ok=True)
+
+        parsed_args.output = pathlib.Path.cwd()
```

Comment:
The change itself is fine, but it's surprising to me that no tests failed or needed to be updated? I think we need at least a test for this, and *also* [this spread test](https://github.com/canonical/craft-application/blob/main/tests/spread/testcraft/test-cmd/tests/spread/my-suite/my-task/task.yaml) should check that `CRAFT_ARTIFACT` is a valid file (since this is ultimately the signal that everything was hooked up correctly)